app/routes/contact/components/ContactForm.tsx

```diff
@@ -60,20 +62,27 @@ const ContactForm = (props: Props) => {
         Hovedstyret, enten du har spørsmål, tilbakemeldinger, eller bare ønsker
         å dele informasjon med oss.
       </p>
-      <p>
-        For mer sensitive saker, eller hvis du ønsker å varsle anonymt med
-        mulighet for respons, burde du bruke vår{' '}
-        <a
-          href="https://avvik.abakus.no"
-          rel="noopener noreferrer"
-          target="_blank"
-        >
-          tredjeparts varslingsportal
-        </a>
-        . Dette sikrer i større grad at du kan kommunisere din bekymring uten å
-        avsløre din identitet, samtidig som det gir oss mulighet til å håndtere
-        situasjonen på en effektiv måte.
-      </p>
+      <Card info>
+        <p>
+          Dersom du ønsker å varsle om kritikkverdige forhold, vennligst benytt
```

Comment:
"kritikkverdige forhold" seems kinda off. Would you call sexual harassment "kritikkverdig"?